### DISCUSSION TITLE
Add delegation events to activity feed

### DISCUSSION THREAD

**Discussion by oliverbarnes** [liquidvotingio/decidim-module-liquidvoting#46] [Ideas]
This feed that's also partly shown on Decidim's main page: https://meta.decidim.org/last_activities

**Comment by oliverbarnes** [comment]
Some delegations make sense to make public (endorsements), others don't (supports)

**Comment by davefrey** [comment]
I think it's a good exercise:
* it's a functionality of decidim, let's step up and be more fully decidim competent
* it pushes our thinking about how delegations are like or unlike votes (like the public/private that you mention)

That said, looking at the meta link, I don't see anything like votes or endorsements, I guess they'd be under Proposals, or maybe they should have their own leftnav radio button.

That interface seems clunky to me, it's slow, it doesn't remember my per-page choice, and to me, those types should be checkboxes not radio choices.

So it could be an opportunity also to implement for us (votes and delegations could get first-class treatment in our version), and give a PR for the general part of what we do back to decidim core

**Reply by davefrey** [reply]
I should add: I think we should only invest here if we intend a solution that we're proud of and want others (demo, sandbox, pilot participants) to see.
